vector/src/main/java/im/vector/app/core/resources/BuildMeta.kt

```diff
@@ -16,8 +16,16 @@
 
 package im.vector.app.core.resources
 
-import android.os.Build
-
 data class BuildMeta(
-        val sdkInt: Int = Build.VERSION.SDK_INT
+        val isDebug: Boolean,
```

Comment:
Do we really need to propagate `isDebug` to submodules? I think if we chose the `Debug` (resp `Release`) `buildType`, all the submodules will use the same `buildType`, and so all `BuildConfig.DEBUG` will have the same values for all modules.

Reply:
Good points. Thanks!